docs/sources/overview/_index.md

```diff
@@ -2,31 +2,79 @@
 title: Overview
 weight: 150
 ---
-# Overview of Loki
+# Overview
 
-Grafana Loki is a set of components that can be composed into a fully featured
-logging stack.
+Grafana Loki is a log aggregation tool,
+and it is the core of a fully-featured logging stack.
 
-Unlike other logging systems, Loki is built around the idea of only indexing
-labels for logs and leaving the original log message unindexed. This means
-that Loki is cheaper to operate and can be orders of magnitude more efficient.
+Loki is a key-value datastore optimized for efficiently holding log data.
```

Comment:
I don't think we should say we're a key-value store; I'm not sure what it means in this context.
```suggestion
Loki is a datastore optimized for efficiently holding log data.
```